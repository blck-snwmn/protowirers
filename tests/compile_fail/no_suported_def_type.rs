use protowirers::*;

#[derive(Proto)]
struct Sample {
    #[def(field_num = 1, def_type = "xint32")]
    s: u32,
}

fn main() {}
