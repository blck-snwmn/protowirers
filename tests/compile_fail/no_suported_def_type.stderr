error: no suport def_type. got=`xint32`.
 --> $DIR/no_suported_def_type.rs:5:37
  |
5 |     #[def(field_num = 1, def_type = "xint32")]
  |                                     ^^^^^^^^
